back-end/src/main/java/org/antop/billiardslove/jpa/entity/Contest.java

```diff
@@ -222,6 +223,7 @@ public boolean isEnd() {
      * 참가자 수 증가
      */
     public void incrementJoiner() {
+        if (currentJoiner == maxJoiner) throw new ContestMaxJoinerException(maxJoiner);
```

Comment:
```suggestion
        if (currentJoiner > maxJoiner) {
          throw new ContestMaxJoinerException(maxJoiner);
        }
```

- 동시성에 대한 고려가 부족한 경우 currentJoiner 가 maxJoiner 보다 커지는 경우가 생길 수 있습니다. 로직상 같은 의미라면 좀 더 안전한 코딩을 추천합니다.
- 괄호는 생략하지 않습니다.
![image](https://user-images.githubusercontent.com/55722186/148151200-fbe8c4b3-2913-4853-adf7-6f91af38c642.png)

Reply:
1. 현재 대회 한건을 조 회시에 비관적락을 사용해서(for update) 조회하고는 있습니다. 안전한 코딩으로 `>=` 👍. 더 생각해보니 한건 조회도 락을 잡는 조회와 안잡는 조회를 나눠야 겠네요.. 
2. 저도 여태까지는 브레이서를 무조건 썻었었는데 요즘 다른 코드를 보면서 `if`문 다음에 한 줄로 짧게 쓸 수 있는 `return` `throw` 문 정도는 브레이서 없이 오른쪽 작성하는 스타일을 시도중(?) 입니다.